### PR TITLE
Fixed warning CircularBiffer.h to CircularBiffer.hpp

### PR DESCRIPTION
**Pull request by faizannazir**
change
 #include <CircularBuffer.h> to
#include <CircularBuffer.hpp>

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -12,10 +12,10 @@
 src_dir = examples/full
 lib_dir = .
 default_envs =
-	esp32dev_nimble
-	esp32dev_native
-	esp32dev
-	nano_33_iot
+	esp32dev_nimble          ;recommended
```
Comment (vovagorodok) [inline platformio.ini:15]: Why additional envs are commented?

**Comment by vovagorodok** [comment]
Will be nice to update it https://github.com/vovagorodok/ArduinoBleOTA/blob/73b261c2317661f8452eb9039031b6e488181057/library.json#L31
to 1.4.0

File: platformio.ini (hunk 1 above)
Comment (faizannazir) [reply platformio.ini:15]: For flashing my board i commented them it was runing all environment. I  am new platformio didn't now if i can run 1 env using command or not

Comment (vovagorodok) [reply platformio.ini:15]: Understand. Just click here:
image
and chose your env

**Comment by vovagorodok** [comment]
Meged

**Pull request closed without merging**